modular_skyrat/modules/autotransfer/code/autotransfer.dm

```diff
@@ -29,7 +29,7 @@ SUBSYSTEM_DEF(autotransfer)
 	curvotes = SSautotransfer.curvotes
 
 /datum/controller/subsystem/autotransfer/fire()
-	if(REALTIMEOFDAY < targettime)
+	if(REALTIMEOFDAY < targettime || !SSticker.current_state == GAME_STATE_PLAYING || EMERGENCY_AT_LEAST_DOCKED)
```

Comment:
```suggestion
	if(REALTIMEOFDAY < targettime || !SSticker.current_state == GAME_STATE_PLAYING || EMERGENCY_AT_LEAST_DOCKED)
		return
	if(!isnull(SSvote.current_vote)
```

Reply:
This also bugs me, cancelling an ongoing vote (such as map vote)

The subsystem refires so just wait.